src/Pages/URLShortener/URLShortener.js

```diff
@@ -27,8 +27,13 @@ export default function URLShortenerPage(props) {
   const [toggleDelete, setToggleDelete] = useState(false);
   const [currentSortColumn, setCurrentSortColumn] = useState(null);
   const [currentSortOrder, setCurrentSortOrder] = useState(null);
+  const [expireDate, setExpireDate] = useState('');
+  const [expirationMessage, setExpirationMessage] = useState('');
 
-  const INPUT_CLASS = 'indent-2 block w-full rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6 text-white';
+  const date = new Date();
+  const minDate = date.toISOString().split('T')[0];
```

Comment:
is the min date tomorrow at midnight? can we do this https://stackoverflow.com/a/15790005

Reply:
```suggestion
  const currentDate = new Date();
  const nextDate = new Date(currentDate);
  nextDate.setDate(currentDate.getDate() + 1);
  nextDate.setHours(0, 0, 0, 0);
  const minDate = nextDate.toISOString().split('T')[0];
```

Now the min date is tomorrow at midnight